lib/utils/events.js

```diff
@@ -32,6 +32,22 @@ module.exports = function (rl) {
 
     spaceKey: keypress.filter(function (e) {
       return e.key && e.key.name === 'space';
+    }).share(),
+
+    allKey: keypress.filter(function (e) {
```

Comment:
All these names are not really reusable. I'd prefer these observable to be called `aKey`, `iKey`, etc.

We might want to listen to `a` keypress, but it might have another meaning than "select all" in other contexts.